packages/create-react-native-library/src/index.ts

```diff
@@ -73,9 +85,15 @@ async function create(_argv: yargs.Arguments<Args>) {
   assertUserInput(questions, answers);
 
   const bobVersion = await resolveBobVersion();
+  const nitroModulesVersion = await resolveNitroModulesVersion();
+  const nitroCodegenVersion = await resolveNitroCodegenVersion();
```

Comment:
if user didn't select nitro then we shouldn't wait for it. when not in interactive mode it can add up to 2 second delay to the CLI